oci.go

```diff
@@ -67,3 +67,35 @@ func validCreateParams(containerID, bundlePath string) error {
 
 	return nil
 }
+
+func validContainer(containerID string) error {
+	// container ID MUST be provided.
+	if containerID == "" {
+		return fmt.Errorf("Missing container ID")
+	}
+
+	// container ID MUST exist.
+	exist, err := containerExists(containerID)
+	if err != nil {
+		return err
+	}
+	if exist == false {
+		return fmt.Errorf("Container ID does not exist")
+	}
+
+	return nil
+}
+
+func processRunning(pid int) (bool, error) {
```

Comment:
The more idiomatic way to search for a process is to send it signal 0. Something like:

```Go
process, err := os.FindProcess(int(pid))
if err != nil {
        return err
} else {
        if err := process.Signal(syscall.Signal(0)) ; err != nil {
                    return err
        }
            
        return nil
}
```